Source/Core/Core/IOS/ES/ES.cpp

```diff
@@ -583,6 +584,13 @@ IPCCommandResult ES::AddContentData(const IOCtlVRequest& request)
   return GetDefaultReply(IPC_SUCCESS);
 }
 
+static bool CheckContentHash(const std::vector<u8>& content, const IOS::ES::Content& info)
```

Comment:
`CheckIfContentHashMatches` is probably more indicative of what checking behavior is being done.